src/main/kotlin/no/nav/syfo/persistering/db/PersisterDialogmeldingQueries.kt

```diff
@@ -99,6 +99,20 @@ fun Connection.erDialogmeldingOpplysningerLagret(dialogmeldingid: String) =
         }
     }
 
+fun Connection.DialogmegldingDokumentByShaString(shaString: String) =
```

Comment:
Skriveleif: Dialogmeglding  -> Dialogmelding

Reply:
Burde funksjonsnavnet starte med et verb? getDialogmelding/hentDialogmelding elns?

Reply:
Jeg prøver å unngå det. Jeg tror det var Fred George som sa at det er unødvendig, fordi det er implisitt. Men her er vel kanskje metodenavnet feil, for vi returnerer ikke egentlig et objekt, vi abre får en boolean som sier om noe ble returnert eller ikke 🤔